modules/openapi-generator/src/main/resources/python/python-experimental/model_templates/method_init_simple.mustache

```diff
@@ -8,30 +8,39 @@
     ])
 
     @convert_js_args_to_python_args
-    def __init__(self{{#hasRequired}}, value{{/hasRequired}}, *args, **kwargs):
+    def __init__(self, *args, **kwargs):
         """{{classname}} - a model defined in OpenAPI
 
-{{#hasRequired}}
+        Note that value can be passed either in args or in kwargs, but not in both.
+
         Args:
             value ({{{dataType}}}):{{#description}} {{description}}.{{/description}}{{#defaultValue}} if omitted the server will use the default value of {{{defaultValue}}}{{/defaultValue}}{{#allowableValues}}, must be one of [{{#enumVars}}{{{value}}}, {{/enumVars}}]{{/allowableValues}}  # noqa: E501
-
-{{/hasRequired}}
         Keyword Args:
-{{^hasRequired}}
             value ({{{dataType}}}):{{#description}} {{description}}.{{/description}}{{#defaultValue}} defaults to {{{defaultValue}}}{{/defaultValue}}{{#allowableValues}}, must be one of [{{#enumVars}}{{{value}}}, {{/enumVars}}]{{/allowableValues}}  # noqa: E501
-{{/hasRequired}}
 {{> python-experimental/model_templates/docstring_init_required_kwargs }}
         """
 
-{{^hasRequired}}
+        value = None
         if 'value' in kwargs:
             value = kwargs.pop('value')
         elif args:
             args = list(args)
             value = args.pop(0)
+{{#defaultValue}}
         else:
             value = {{{defaultValue}}}
+{{/defaultValue}}
+{{^defaultValue}}
+{{#hasRequired}}
```

Comment:
Please remove this hasRequired block.
In this case there is no default value. There must always be an input value for modelSimple models. We don't need to filter by hasRequired any more.